unix/src/machine_stats/__init__.py

```diff
@@ -235,26 +241,6 @@ def run(self):
             # Remove ansible tmpdir
             shutil.rmtree(C.DEFAULT_LOCAL_TMP, True)  # pylint: disable=no-member
 
-            eprint("UP *********************************")
```

Comment:
ah, I get it, we are removing these prints and loops since we no longer have those Global variables you told me about.